src/bindings/python/src/openvino/runtime/__init__.py

```diff
@@ -44,6 +44,7 @@
 from openvino._pyopenvino import get_batch
 from openvino._pyopenvino import set_batch
 from openvino._pyopenvino import serialize
+from openvino._pyopenvino import shutdown
```

Comment:
```suggestion
from openvino._pyopenvino import shutdown
```
Is it supposed to be exposed to users via `openvino.runtime` module? Is this something that will be (1) used every time by the user or (2) is it only necessary for FEs development?

If (1): we need to discuss the namespace.
If (2): It can be obtained manually from `_pyopenvino` or put into another namespace (maybe specific to paddle FE?).

Reply:
Discussed offline